README.md

```diff
@@ -24,7 +24,8 @@ appropriate one:
 |------------------|-------------------|---------------|----------------|
 | Lilac            | `>=12.0, <13`     | Not supported | Not supported  |
 | Maple            | `>=13.2, <14`[^1] | `maple`       | 0.3.x          |
-| Nutmeg           | `>=14.0, <15`     | `main`        | 1.x.x          |
+| Nutmeg           | `>=14.0, <15`     | `nutmeg`      | 1.x.x          |
+| Olive            | `>=15.0, <16`     | `main`        | 2.x.x          |
```

Comment:
We also don't need `nutmeg` branch and for Olive also version `1.x.x` then :)

Reply:
Fair. :) Also fixed.